app/controllers/settings/dex_connector_oidcs_controller.rb

```diff
@@ -0,0 +1,94 @@
+# Settings::DexConnectorOidcsController is responsible to manage requests
+# related to OIDC connectors.
+class Settings::DexConnectorOidcsController < SettingsController
+  # populate @data_holder with existing database record
+  before_action :set_data_holder, except: [:index, :new, :create]
+
+  def index
+    @oidc_connectors = DexConnectorOidc.all
+  end
+
+  def new
+    @is_data_valid = false
+    @data_holder = data_holder_type.new
+  end
+
+  def edit
+    @is_data_valid = false
```

Comment:
This is more of a personal preference but if we omit `@is_data_valid`, whenever we refer it in the templates it would be already a falsy value.

Reply:
I'll have to stare at that one to see if it makes the code significantly cleaned; even after years of Perl, I really don't like depending on implicit initialization. _Maybe_. :)

Reply:
Looking at it, I'm not quite sold on this one.  The var is explicitly set to the result of`.valid?` in create and update, so I think in this specific case I like the consistency a little bit better when it's being set for all four views that use it.  I'm not completely against taking the two in `new` and `edit` out if you're absolutely adamant about making things as compact as possible, though.